### PR TITLE
PTX-4768 Automate Volume Clone

### PR DESCRIPTION
**Pull request by aravindranath14**
**What this PR does / why we need it**:
Automated Volume Clone functionality tests to longevity

**Which issue(s) this PR fixes** (optional)
Closes # PTX-4768

**Special notes for your reviewer**:
LOGS:

STEP: Register test triggers
time="2022-01-31 17:54:50" level=info msg="Finished registering triggers"
time="2022-01-31 17:56:50" level=info msg="Waiting for lock for trigger [volumeClone]\n"
time="2022-01-31 17:56:50" level=info msg="Successfully taken lock for trigger [volumeClone]\n"
STEP: get volumes for all apps in test and clone them
STEP: get volumes for fio app
STEP: Clone Volume fio-data-fio-0
time="2022-01-31 17:56:51" level=info msg="Calling CloneVolume()..."
time="2022-01-31 17:57:12" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:57:12" level=info msg="Getting new driver."
time="2022-01-31 17:57:14" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:57:16" level=info msg="successfully clone pvc-d3c33861-1127-4df1-b011-cef54037db8d as 22072248949087880"
STEP: Validate successful clone 22072248949087880
time="2022-01-31 17:57:37" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:57:37" level=info msg="Getting new driver."
time="2022-01-31 17:57:39" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:57:40" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 22072248949087880
time="2022-01-31 17:58:00" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:58:00" level=info msg="Getting new driver."
time="2022-01-31 17:58:03" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:58:07" level=info msg="successfully deleted Portworx volume 22072248949087880 "
STEP: Clone Volume fio-data-fio-1
time="2022-01-31 17:58:07" level=info msg="Calling CloneVolume()..."
time="2022-01-31 17:58:28" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:58:28" level=info msg="Getting new driver."
time="2022-01-31 17:58:30" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:58:35" level=info msg="successfully clone pvc-926a0d77-11fe-4738-8a14-55d7e6f5dba5 as 911295394185866608"
STEP: Validate successful clone 911295394185866608
time="2022-01-31 17:58:55" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:58:55" level=info msg="Getting new driver."
time="2022-01-31 17:58:57" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:58:58" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 911295394185866608
time="2022-01-31 17:59:19" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:59:19" level=info msg="Getting new driver."
time="2022-01-31 17:59:20" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:59:25" level=info msg="successfully deleted Portworx volume 911295394185866608 "
STEP: Clone Volume fio-data-fio-2
time="2022-01-31 17:59:25" level=info msg="Calling CloneVolume()..."
time="2022-01-31 17:59:46" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 17:59:46" level=info msg="Getting new driver."
time="2022-01-31 17:59:48" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 17:59:58" level=info msg="successfully clone pvc-180417c7-8d31-401b-975a-9ffcf64b89f8 as 44192449145922208"
STEP: Validate successful clone 44192449145922208
time="2022-01-31 18:00:19" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:00:19" level=info msg="Getting new driver."
time="2022-01-31 18:00:20" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:00:21" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 44192449145922208
time="2022-01-31 18:00:42" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:00:42" level=info msg="Getting new driver."
time="2022-01-31 18:00:44" level=info msg="Using 10.13.9.115:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:00:49" level=info msg="successfully deleted Portworx volume 44192449145922208 "
STEP: Clone Volume fio-data-fio-3
time="2022-01-31 18:00:49" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:01:09" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:01:09" level=info msg="Getting new driver."
time="2022-01-31 18:01:11" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:01:23" level=info msg="successfully clone pvc-ed934b46-e76b-4a8c-a2cd-4f1e3493df85 as 271772208475696650"
STEP: Validate successful clone 271772208475696650
time="2022-01-31 18:01:43" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:01:43" level=info msg="Getting new driver."
time="2022-01-31 18:01:45" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:01:46" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 271772208475696650
time="2022-01-31 18:02:07" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:02:07" level=info msg="Getting new driver."
time="2022-01-31 18:02:09" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:02:15" level=info msg="successfully deleted Portworx volume 271772208475696650 "
STEP: Clone Volume fio-data-fio-4
time="2022-01-31 18:02:15" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:02:36" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:02:36" level=info msg="Getting new driver."
time="2022-01-31 18:02:38" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:02:51" level=info msg="successfully clone pvc-9317a322-bd31-4f95-a0ae-7e110eaa5e87 as 828935785200982730"
STEP: Validate successful clone 828935785200982730
time="2022-01-31 18:03:12" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:03:12" level=info msg="Getting new driver."
time="2022-01-31 18:03:14" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:03:15" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 828935785200982730
time="2022-01-31 18:03:35" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:03:35" level=info msg="Getting new driver."
time="2022-01-31 18:03:37" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:03:45" level=info msg="successfully deleted Portworx volume 828935785200982730 "
STEP: Clone Volume fio-data-fio-5
time="2022-01-31 18:03:45" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:04:05" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:04:05" level=info msg="Getting new driver."
time="2022-01-31 18:04:07" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:04:18" level=info msg="successfully clone pvc-2a1bbf5e-4743-48ec-9eee-b255e2f115bc as 450302805937893703"
STEP: Validate successful clone 450302805937893703
time="2022-01-31 18:04:39" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:04:39" level=info msg="Getting new driver."
time="2022-01-31 18:04:41" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:04:41" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 450302805937893703
time="2022-01-31 18:05:02" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:05:02" level=info msg="Getting new driver."
time="2022-01-31 18:05:04" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:05:12" level=info msg="successfully deleted Portworx volume 450302805937893703 "
STEP: Clone Volume fio-log-fio-0
time="2022-01-31 18:05:12" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:05:33" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:05:33" level=info msg="Getting new driver."
time="2022-01-31 18:05:34" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:05:48" level=info msg="successfully clone pvc-9186b524-af55-45bf-86b6-62b1e9926ae9 as 827098413446930902"
STEP: Validate successful clone 827098413446930902
time="2022-01-31 18:06:09" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:06:09" level=info msg="Getting new driver."
time="2022-01-31 18:06:11" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:06:11" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 827098413446930902
time="2022-01-31 18:06:32" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:06:32" level=info msg="Getting new driver."
time="2022-01-31 18:06:34" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:06:43" level=info msg="successfully deleted Portworx volume 827098413446930902 "
STEP: Clone Volume fio-log-fio-1
time="2022-01-31 18:06:43" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:07:04" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:07:04" level=info msg="Getting new driver."
time="2022-01-31 18:07:05" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:07:19" level=info msg="successfully clone pvc-8fe516ed-cdd6-4507-a490-5e1565872d1e as 736458289967036456"
STEP: Validate successful clone 736458289967036456
time="2022-01-31 18:07:40" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:07:40" level=info msg="Getting new driver."
time="2022-01-31 18:07:41" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:07:42" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 736458289967036456
time="2022-01-31 18:08:03" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:08:03" level=info msg="Getting new driver."
time="2022-01-31 18:08:05" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:08:11" level=info msg="successfully deleted Portworx volume 736458289967036456 "
STEP: Clone Volume fio-log-fio-2
time="2022-01-31 18:08:11" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:08:32" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:08:32" level=info msg="Getting new driver."
time="2022-01-31 18:08:34" level=info msg="Using 10.13.9.115:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:08:52" level=info msg="successfully clone pvc-b0072785-ec3d-49fd-af13-cb8686f7996d as 308435817378023186"
STEP: Validate successful clone 308435817378023186
time="2022-01-31 18:09:13" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:09:13" level=info msg="Getting new driver."
time="2022-01-31 18:09:15" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:09:15" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 308435817378023186
time="2022-01-31 18:09:36" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:09:36" level=info msg="Getting new driver."
time="2022-01-31 18:09:38" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:09:47" level=info msg="successfully deleted Portworx volume 308435817378023186 "
STEP: Clone Volume fio-log-fio-3
time="2022-01-31 18:09:47" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:10:08" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:10:08" level=info msg="Getting new driver."
time="2022-01-31 18:10:10" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:10:31" level=info msg="successfully clone pvc-969182c0-7b14-4d35-bcaa-58b98bc0a396 as 1185680397083201"
STEP: Validate successful clone 1185680397083201
time="2022-01-31 18:10:52" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:10:52" level=info msg="Getting new driver."
time="2022-01-31 18:10:53" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:10:54" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 1185680397083201
time="2022-01-31 18:11:15" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:11:15" level=info msg="Getting new driver."
time="2022-01-31 18:11:16" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:11:29" level=info msg="successfully deleted Portworx volume 1185680397083201 "
STEP: Clone Volume fio-log-fio-4
time="2022-01-31 18:11:29" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:11:50" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:11:50" level=info msg="Getting new driver."
time="2022-01-31 18:11:52" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:12:14" level=info msg="successfully clone pvc-e7985bdc-dae8-41d9-b9d6-0be1d5246367 as 890086842815677597"
STEP: Validate successful clone 890086842815677597
time="2022-01-31 18:12:35" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:12:35" level=info msg="Getting new driver."
time="2022-01-31 18:12:36" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:12:37" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 890086842815677597
time="2022-01-31 18:12:58" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:12:58" level=info msg="Getting new driver."
time="2022-01-31 18:13:00" level=info msg="Using 10.13.9.132:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:13:36" level=info msg="successfully deleted Portworx volume 890086842815677597 "
STEP: Clone Volume fio-log-fio-5
time="2022-01-31 18:13:36" level=info msg="Calling CloneVolume()..."
time="2022-01-31 18:13:57" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:13:57" level=info msg="Getting new driver."
time="2022-01-31 18:13:58" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:14:26" level=info msg="successfully clone pvc-dd560c9f-f54c-4c73-a12b-03adca38b38b as 478782630535076765"
STEP: Validate successful clone 478782630535076765
time="2022-01-31 18:14:47" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:14:47" level=info msg="Getting new driver."
time="2022-01-31 18:14:49" level=info msg="Using 10.13.9.115:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:14:49" level=debug msg="Validating Stork snapshot"
STEP: cleanup the cloned volume 478782630535076765
time="2022-01-31 18:15:10" level=info msg="testAndSetEndpoint failed for 10.233.46.123: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 10.233.46.123:9020: i/o timeout\""
time="2022-01-31 18:15:10" level=info msg="Getting new driver."
time="2022-01-31 18:15:12" level=info msg="Using 10.13.9.131:9020 as endpoint for portworx volume driver"
time="2022-01-31 18:16:11" level=info msg="successfully deleted Portworx volume 478782630535076765 "
time="2022-01-31 18:16:11" level=info msg="Trigger Function completed for [volumeClone]\n"
time="2022-01-31 18:16:11" level=info msg="Successfully released lock for trigger [volumeClone]\n"

### CODE REVIEW

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -328,6 +328,62 @@ func (d *portworx) isMetadataNode(node node.Node, address string) (bool, error)
...
+		return "", fmt.Errorf("failed to find volume %v due to %v", volumeID, err)
+	}
+	pxVolume := volumeInspectResponse.Volume
+
+	volID := pxVolume.Id
+	cloneVolumeName := pxVolume.Locator.Name + "_clone"
+
+	volumeCloneResp, err := volDriver.Clone(d.getContext(), &api.SdkVolumeCloneRequest{ParentId: volID, Name: cloneVolumeName})
+	if err != nil {
+		logrus.Infof("Error %v", err)
```
Comment (lsrinivas-pure) [inline drivers/volume/portworx/portworx.go:344]: you can remove this log statement as the error is getting printed in next line

File: drivers/volume/portworx/portworx.go [hunk 2]
```diff
@@ -328,6 +328,62 @@ func (d *portworx) isMetadataNode(node node.Node, address string) (bool, error)
...
+func (d *portworx) DeleteVolume(volumeID string) error {
+	volDriver := d.getVolDriver()
+	volumeInspectResponse, err := volDriver.Inspect(d.getContext(), &api.SdkVolumeInspectRequest{VolumeId: volumeID})
+	if err != nil {
+		return fmt.Errorf("failed to find volume %v due to %v", volumeID, err)
+	}
+
+	pxVolume := volumeInspectResponse.Volume
+	volID := pxVolume.Id
+	deleteVolResp, err := volDriver.Delete(d.getContext(), &api.SdkVolumeDeleteRequest{VolumeId: volID})
```
Comment (lsrinivas-pure) [inline drivers/volume/portworx/portworx.go:370]: you can put _ here itself if you do not need the variable

**Review by lsrinivas-pure — COMMENTED** [review]
Also please run the regressions tests using tp-dev job

File: drivers/volume/portworx/portworx.go (hunk 1 above)
Comment (aravindranath14) [reply drivers/volume/portworx/portworx.go]: Removed.

File: drivers/volume/portworx/portworx.go (hunk 2 above)
Comment (aravindranath14) [reply drivers/volume/portworx/portworx.go]: Used _ in the first instance, removed the variable

**Comment by aravindranath14** [comment]
> 

Here is a successful regression run: https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/984/

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**